src/ontology/uberon-edit.obo

```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
 property_value: dcterms-date 2023-02-10T14:54:06Z xsd:dateTime
 
 [Term]
-id: UBERON:8480047
-name: retroauricular region
-def: "The retroauricular region is the area located behind the auricle of the outer ear." [http://orcid.org/0000-0002-0819-0473]
-synonym: "retro-auricular area" EXACT [PMID:22592943]
-synonym: "retro-auricular part of head" EXACT [FMA:322167]
-synonym: "retro-auricular region" EXACT [PMID:33988139]
-synonym: "retroauricular area" EXACT [PMID:29470606]
-is_a: UBERON:0000061 ! anatomical structure
-relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
-relationship: part_of UBERON:0000033 ! head
-property_value: curator_notes "Exists at least in all mammals. https://www.imaios.com/en/vet-Anatomy/Vet-Anatomical-Part/Retroauricular-region" xsd:string
-property_value: dcterms-date 2023-02-09T09:39:18Z xsd:dateTime
+id: UBERON:8600004
+name: visceral muscle tissue
+def: "A muscle tissue that is associated with internal organs (viscera), except the heart." [BTO:0002780, CLAO:0001295]
+comment: It is quite common to refer to visceral muscles as smooth muscles, by analogy with vertebrates; however most visceral muscles in arthropods are actually striated. {xref="PMID:12132390", xref="PMID:8720463"}
+is_a: UBERON:0002385 ! muscle tissue
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-16T10:17:46Z xsd:dateTime
+
+[Term]
+id: UBERON:8600005
+name: visceral smooth muscle tissue
+def: "A smooth muscle tissue that is associated with internal organs (viscera)." [PMID:12132390, PMID:33729000]
+is_a: UBERON:0001135 ! smooth muscle tissue
+is_a: UBERON:8600004 ! visceral muscle tissue
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-16T10:32:12Z xsd:dateTime
+
+[Term]
+id: UBERON:8600006
+name: visceral striated muscle tissue
+def: "A striated muscle tissue that is associated with internal organs (viscera), except the heart." [https://pressbooks.pub/rbtallitsch/chapter/chapter-6-muscle-tissue/, PMID:8720463]
+synonym: "striated visceral muscle tissue" EXACT []
+is_a: UBERON:0002036 ! striated muscle tissue
+is_a: UBERON:8600004 ! visceral muscle tissue
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-16T10:34:43Z xsd:dateTime
+
+[Term]
+id: UBERON:8600007
+name: transversely striated visceral muscle tissue
+def: "A striated visceral muscle tissue that has a transverse striation." [PMID:8720463]
```

Comment:
From the dbxref:
"The muscular cells of invertebrates can be divided into three major classes on the basis of their striation pattern: **transversely striated, obliquely striated**, or smooth muscle."

Is 'visceral striated muscle tissue' meant to be a grouping term for 'transversely striated visceral muscle tissue' and 'obliquely striated visceral muscle tissue'? If so, will you be adding 'obliquely striated visceral muscle tissue'?

Alternatively, consider modelling 'visceral striated muscle tissue' as 'obliquely striated visceral muscle tissue'.

Reply:
Thank you @bvarner-ebi. From the dbxref:

"Although all the arthropod muscles present a common ultrastructural pattern, which may be described as `transversely striated muscle` with continuous Z lines, there are dissimilarities among zoological groups, and even within the same species, in relation to the anatomical type of muscle. Two anatomical types of muscles, specialised for different functions, may be distinguished in arthropods: skeletal (somatic) muscles, `and visceral muscles (digestive tract and hemolymphatic vessels)`. Another type of muscle should be added in insects: the flight muscles, which may either be of the direct type (odonates, lepidopteran and orthopteran) or the indirect type (dipteran, hymenopteran, coleopteran and hemipteran)."

Meaning that there are three different transversely striated muscle, one of them being visceral. However, although not mention for arthropods, in annelids there is an `obliquely striated muscle` associated with the intestinal wall:

"Recently, this [Obliquely striated] muscle type has also been described in the intestinal wall (Fig. 7a,b) and pseudohearts (Fig. 7c,d) of the oligochaete E.foetida (Royuela et al., 1995)."

Visceral obliquely striated muscle are also [found in snails](https://doi.org/10.1007/bf00227599). Interestingly, in this reference there are cross striated muscles, which might be a synonym for transversely striated muscles.